integration-tests/src/test/resources/features/action-sendtobridge.feature

```diff
@@ -0,0 +1,105 @@
+Feature: SendToBridge Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "bridge1"
+    And the Bridge "bridge1" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "bridge1" is available within 2 minutes
+
+  Scenario: Send Cloud Event to SendToBridge with bridgeId defined
+    Given create a new Bridge "bridge2"
+    And the Bridge "bridge2" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "bridge2" is available within 2 minutes
+
+    And add a Processor to the Bridge "bridge1" with body:
+    """
+    {
+      "name": "sendToBridgeWithBridgeIdProcessor",
+      "action": {
+        "parameters": {
+            "bridgeId":  "${bridge.bridge2.id}"
+        },
+        "type": "SendToBridge"
+      }
+    }
+    """
+    And the Processor "sendToBridgeWithBridgeIdProcessor" of the Bridge "bridge1" is existing with status "ready" within 3 minutes
```

Comment:
Is there any particular reason for using 3 minutes timeout instead of 2 minutes?

Reply:
all processor creation steps in the features are using the 3 minutes